MAUI/Scheduler/drag-and-drop-appointments.md

```diff
@@ -0,0 +1,191 @@
+`SourceResource` - Gets the original resource of the appointment that is being dropped.
+`TargetResource` - Gets the resource into which the appointment is being dropped.
+
+{% tabs %}
+{% highlight c# %}
+scheduler.AppointmentDrop += OnSchedulerAppointmentDrop;
+
+...
+
+private void OnSchedulerAppointmentDrop(object? sender, AppointmentDropEventArgs e)
+{
+        var appointment = e.Appointment;
+        e.Cancel = false;
+        var dropTime = e.DropTime;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing the Drag and Drop environment
+You can handle the behavior of drag and drop in Scheduler by using `DragDropSettings` property.
+
+{% tabs %}
+{% highlight xaml %}
+<schedule:SfScheduler x:Name="dragAndDrop" ShowBusyIndicator="True"
+                    AppointmentsSource="{Binding DragEvents}"
+                    View="Week"
+                    DisplayDate="{Binding DisplayDate}"
+                    AllowedViews="Day,Week,WorkWeek,Month">
+        <schedule:SfScheduler.DragDropSettings>
+            <schedule:DragDropSettings AllowNavigation="true" AllowScroll="true" 
+                                    ShowTimeIndicator="true"/>
+        </schedule:SfScheduler.DragDropSettings>
+</schedule:SfScheduler>
+{% endhighlight %}
+{% highlight c# %}
+scheduler.DragDropSettings = new DragDropSettings 
+{
+    AllowNavigation = true,
+    AllowScroll = true,
+    ShowTimeIndicator = true,
+};
+{% endhighlight %}
+{% endtabs %}
+
+#### Disabling navigation when dragging appointment
+Using `AllowNavigate` property of `DragDropSettings` you can handle whether you can navigate views while dragging the appointment or not. The default value of the AllowNavigate property is `true`.
+
+{% tabs %}
+{% highlight c# %}
+scheduler.DragDropSettings.AllowNavigation = true;
+{% endhighlight %}
+{% endtabs %}
+
+#### Handling navigation delay while holding dragged appointment
+Using `AutoNavigationDelay` property you can handle the navigation time when navigating through views while holding the dragged appointment.
+{% tabs %}
+{% highlight c# %}
+scheduler.DragDropSettings.AutoNavigationDelay = new TimeSpan(0, 0, 0, 1, 0);
+{% endhighlight %}
+{% endtabs %}
+
+#### Disabling scroll when dragging appointment
+Using `AllowScroll` property you can handle the scrolling the time slot while dragging the appointment or not. while dragging the appointment to the endpoint of the current view in Schedule. The default value of the `AllowScroll` property is `true`.
+
+{% tabs %}
+{% highlight c# %}
+scheduler.DragDropSettings.AllowScroll = true;
+{% endhighlight %}
+{% endtabs %}
+
+#### Disabling dragging time indicator
+You can handle whether the drag and drop indicator should be displayed within the scheduler during appointment dragging by using `ShowTimeIndicator` property which shows the dragged appointment current position time. The default value of the `ShowTimeIndicator` property is `true`.
```

Comment:
Add Note for NA views , will not display on time ruler width zero

https://help.syncfusion.com/wpf/scheduler/appointment-drag-and-drop

Reply:
Added note